index.html

```diff
@@ -305,7 +305,14 @@ <h2 class="tittle">Meet our experts</h2>
                   <img class="wow fadeInUp img-responsive img-thumbnail" src="http://res.cloudinary.com/dews/image/upload/c_scale,w_235/v1469466059/Clinic/dr.ankur.jpg">
                   <div class="team-address">
                     <p>Dr. Ankur Mishra</p>
-                    <span>BHMS, PGDCC</span>
+                    <span>BHMS, FHD, PGDCC (Dermatologist)</span>
+                  </div>
+                </li>
+                <li>
+                  <img class="wow fadeInUp img-responsive img-thumbnail" src="https://github.com/aakruti27/aakruti27.github.io/blob/master/IMG-20191201-WA0016.jpg?raw=true" style="visibility: visible; animation-name: fadeInUp;">
+                  <div class="team-address">
```

Comment:
Maybe add the image to this PR also (under path - /assets/images/) ? Then path would be like - `src="assets/images/IMG-20191201-WA0016.jpg"` 

`style="visibility: visible; animation-name: fadeInUp;"` - isn't there in other images, is it needed ?